tests/python_package_test/test_engine.py

```diff
@@ -1959,32 +1963,32 @@ def train_booster(params=params_obj_verbose, **kwargs):
 
     # fobj, no feval
     # no default metric
-    res = get_cv_result(params=params_verbose, fobj=dummy_obj)
+    res = get_cv_result(params=params_dummy_obj_verbose)
     assert len(res) == 0
 
     # metric in params
-    res = get_cv_result(params=params_metric_err_verbose, fobj=dummy_obj)
```

Comment:
Add `dummy_obj` to `params_metric_err_verbose` and rename them into `params_dummy_obj_metric_err_verbose`

Reply:
copy that